memtagabielf64/README.md

```diff
@@ -17,19 +17,18 @@ is licensed under CC-BY-SA-4.0 along with an additional patent
 license.  The language in the additional patent license is largely
 identical to that in Apache-2.0 (specifically, Section 3 of Apache-2.0
 as reflected at https://www.apache.org/licenses/LICENSE-2.0) with two
-exceptions.
-
-First, several changes were made related to the defined terms so as to
-reflect the fact that such defined terms need to align with the
-terminology in CC-BY-SA-4.0 rather than Apache-2.0 (e.g., changing
-“Work” to “Licensed Material”).
-
-Second, the defensive termination clause was changed such that the
-scope of defensive termination applies to “any licenses granted to
-You” (rather than “any patent licenses granted to You”).  This change
-is intended to help maintain a healthy ecosystem by providing
-additional protection to the community against patent litigation
-claims.
+exceptions:
+
+- First, several changes were made related to the defined terms so as to reflect
+the fact that such defined terms need to align with the terminology in
```

Comment:
I checked the preview on Github before committing and it looks like this doesn't need to be the case. At least according to https://github.com/ARM-software/abi-aa/blob/b9529223dae10b63fd10758992f5404d0a4ab112/memtagabielf64/README.md it looks like it's indented properly.

Reply:
I looked up the Github markdown spec, and I guess you're right and we better obey the spec, instead of the implementation:

https://github.github.com/gfm/#example-264